aicsimageio/tests/readers/test_tiff_reader.py

```diff
@@ -26,6 +26,13 @@
             np.uint16,
             0,
         ),
+        (
+            "s_3_t_1_c_1_z_1_RGB.tiff",
+            (7548, 7548, 3),
+            "YXS",
+            np.uint16,
+            0,
+        ),
```

Comment:
I love that our tests have become this easy to write.

Reply:
Once we decide on how the dims should be returned there should also be a test added to aicsimage to ensure that the dims get swizzled around properly.